basic-guides/arch-glacor-basic.txt

```diff
@@ -82,60 +82,53 @@ The Arch-Glacor will cycle through sets of 5 different mechanics (Glacyte Minion
 
-⬥ The spawning animation will interrupt the attack animation of the previous auto attack, but the auto attack will still go through. This is especially important to know if the previous mechanic is Flurry.
+⬥ Sometimes, the minion spawn animation can hide Arch-Glacor's attack animations.
 .
 .img:https://i.imgur.com/kSB6UcI.mp4
 .
 
 .
 __**Flurry**__
-⬥ Normal mode: Arch-Glacor's auto attacks will switch between <:Magic:689504724159823906> and <:range:580168050121113623> with high damage in their respective style. The player has increased damage reduction when praying against the correct combat style.
+⬥ Normal mode: Arch-Glacor will deal high damage with either <:Magic:689504724159823906> or <:range:580168050121113623>. Ranged attacks are signified with a puff of green smoke.
 
-⬥ Hard mode: Arch-Glacor gains a melee attack where it swipes one of its arms across the arena, in addition to the ranged and magic attacks from normal mode. The damage from auto attacks scales with enrage.
+⬥ Hard mode: Arch-Glacor gains an additional melee attack where it swipes one of its arms across the arena. The damage from all attacks scales with enrage.
 
 ⬥ In either mode, rotate your deflect curses <:DeflectMage:544195487926845462> <:DeflectRange:544195488317046812> <:DeflectMelee:544195488447201300> accordingly to lessen the damage from each auto-attack.
-    • If comfortable, flick to <:soulsplit:615613924506599497> between each one to gain some healing.
+    • If comfortable, flick to <:soulsplit:615613924506599497> between each one.
 
-*Note 1: the last 2 auto-attacks from this mechanic can carry over into Glacyte Minions.*
-*Note 2: if playing with game sounds, each type of attack has a different audio cue.*
+*Note: this mechanic can carry over briefly into Glacyte Minions.*
 .
 .img:https://i.imgur.com/0CU0qXf.mp4
 .
 
 .
 __**Pillars of Ice**__
-⬥ Normal mode: Arch-Glacor forces the player to the west side of the arena, and calls down elemental beams from the sky. The first beam appears shortly after the Arch-Glacor raises its arms, with the second beam appearing just before the Creeping Ice disappears. Being hit by the beam deals a random amount of hard typeless damage and saps 8% of max Prayer points.
+⬥ Normal mode: Arch-Glacor forces the player to the west side of the arena, and spawns elemental beams. The first beam appears shortly after the Arch-Glacor raises its arms, with the second beam appearing just before the Creeping Ice disappears. Touching beams will deal hard typeless damage and drains prayer.
 
-⬥ Hard mode: the same as normal mode, except a third beam will appear shortly after second, in the middle of the arena. Beams will additionally drain 1% adrenaline for every hit on the player. The damage from beams scale with enrage.
+⬥ Hard mode: an additional third beam will appear shortly after second, in the middle of the arena. Touching beams will deal hard typeless damage (scales with enrage) and drains both adrenaline and prayer.
```

Comment:
Repeated mention of it dealing hard typeless damage, can this not be put into it's own bullet point below?

Reply:
Removing it entirely as per discord convo